### ISSUE TITLE
Help! Have private key and wallet.dat backed up, but no passphrase ...

### ISSUE THREAD

**Issue by roleerob** [firoorg/firo#1008]
### Describe the issue

For the 1st time, since depositing my XZC tokens in 2017, I would like to send some.  But, in attempting it, I am prompted for a passphrase.  I do not have a passphrase.  I have the 12 private-key words and I have always faithfully backed up my wallet.dat file.  But ... No passphrase ...

Please advise on what I need to do.

### Expected behaviour
I would like to be able to send my tokens to another address.

### Actual behaviour
Passphrase required and I do not know it.

### Screenshots.
If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.

### What version of bitcoin-core are you using?
List the version number/commit ID, and if it is an official binary, self compiled or a distribution package such as PPA.

### Machine specs:
- OS: Windows 10 Pro 64-bit
- CPU:  Intel(R) Core(TM) i7-10750H
- RAM: 32.0 GB
- Disk size: 1 TB 
- Disk Type (HD/SDD): SSD

Thanks, in advance, for your help!

**Comment by justanwar** [comment]
12 words seed phrase was only introduced in December 2019. Are you sure it is not from Electrum?

If you have place the backed up wallet.dat into the Firo data directory, it will only ask you for a passphrase if you are trying to send out the funds or sign a message.

Which wallet are you using? Do you have the exact screenshot of the passphrase prompt?

**Comment by roleerob** [comment]
Hmmm.  Dec 2019?  I bought the tokens I have in Sept 2017.  And moved them from Bitttrex to the ZCoin Core wallet in Dec 2017.  Which has "morphed" into Firo Core wallet recently ...

The 12 words I have are from the initial set-up of this wallet in Dec 2017.  As I have no record of a passphrase, not sure I was ever asked to provide one, although I will let you know this was all very new to me then, so I may have managed to simply not get it down.  I at least knew the importance of the private key.  And my understanding these 12 words were used to generate it ...

"it will only ask you for a passphrase if you are trying to send out the funds" ... Correct.  That is what I am trying to do.

The version I am currently running:

![image](https://user-images.githubusercontent.com/31572169/110658381-e1421980-817e-11eb-9618-9976ca2ad3d6.png)

Upon clicking the Send button, the error message I am getting:

![image](https://user-images.githubusercontent.com/31572169/110660332-b1941100-8180-11eb-80b0-9025758de5e4.png)

Now what?  Anything else I can provide to help you help me?

**Comment by justanwar** [comment]
The wallet will not ask for a passphrase unless you have encrypted it before with a passphrase.

You can create a backup of the current wallet.dat with the passphrase and attempt to create a new wallet with the 12 words that you have. This will remove the passphrase requirement since you are restoring, but since we only implemented it in December 2019 it might not work.

To do this, close the QT wallet, move the exisiting wallet.dat somewhere else and start the QT wallet again. The `Create or recover wallet` window will pop up. Choose `Recover existing wallet`, `12 words` and enter the 12 words.

**Comment by roleerob** [comment]
This did not work.  I was going to write my FIRO off, but went and read the only notes I have on this wallet again.  "One more time with enthusiasm ..."

The result was I was able to solve the problem, but give God the credit for the inspiration on how I did it.  As mentioned, these notes were from one of the very first purchases of crypto I made in the beginning.  **_I decided the notes could be interpreted to indicate the 12 words were actually the passphrase_**.  I would never have normally thought of that, as I would never myself create a passphrase like this.  Nonetheless, my notes gave me the idea that is perhaps was I was trying to record, since I clearly did not understand what all these new details meant at the time.

In the 3-½ yrs since, I have many times encountered the need to store a series of words as the means of securing a wallet, as somehow the wallet converts these to a cryptographic string - the private key.

Well, the very first word of the 12 was naïve.  Entering it that way failed. I was again about to give it up, when I noted how in typing this word in MS Word - naive - it is "autocorrected" to naïve.  Hmmm.  Was it possible?  So ... I entered it as naive, followed by the remaining 11 and ... It worked! 😜

What are the odds ...

All that to let you know I am closing this case.   Others may read this some day and have an inspiration of their own.  And, if by some means @justanwar, this gives you some ideas about how to improve the wallet?  All the better ...

Thanks! 👋
